src/inputdaemon.cpp

```diff
@@ -191,6 +192,14 @@ void InputDaemon::refreshJoysticks()
                 SDL_Joystick *sdlStick = SDL_GameControllerGetJoystick(controller);
                 SDL_JoystickID tempJoystickID = SDL_JoystickInstanceID(sdlStick);
 
+                if (SDL_GameControllerHasSensor(controller, SDL_SENSOR_GYRO)) {
+                    SDL_GameControllerSetSensorEnabled(controller, SDL_SENSOR_GYRO, SDL_TRUE);
+                }
+
+                if (SDL_GameControllerHasSensor(controller, SDL_SENSOR_ACCEL)) {
```

Comment:
This function is available since `2.0.14`  
This won't work on most of currently used distributions. Ubuntu 18.04 and 20.04 won't work [link](https://pkgs.org/download/libsdl2-dev)

Reply:
I did'n know that 20.4 does not support sensors in SDL. Luckily, Ubuntu 22.04 will release soon.
I see no alternative to using SDL >=2.0.14 because older version can not access gamecontroller sensors.

Do you think this is a problem since distributions which do not update SDL will not update Antimicrox either?

For your information: When I've implemented everything so that it works I will refactor all that code in the pull request.

Reply:
> I see no alternative to using SDL >=2.0.14

I think disabling this feature on versions lower than 2.0.14 could be a good compromise.  
Without this AntiMicroX would be cot compillable on older distributions.

You could use some macros like the ones used for checking QT 

```c++
#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
```

Reply:
You could use macro `SDL_VERSION_ATLEAST(X,Y,Z)`